src/tcgwars/logic/impl/gen3/Deoxys.groovy

```diff
@@ -1806,7 +1806,7 @@ public enum Deoxys implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               damage 30
-              noWrDamage(10, self)
+              damage 10, self //TODO: Check this not doing W/R
```

Comment:
Where does your thought that this doesn't need to explicitly ignore weakness and resistance come from?

Reply:
...Huh, would've sworn there were some psychic weak to psychic (or [L] weak to [L]) that broke this rule... Apparently not, will revert this (In my defense, last weak-to-self card with an attack like this is from 2015 😝 )